examples/tutorial/basic-5/programs/basic-5/src/lib.rs

```diff
@@ -0,0 +1,69 @@
+// #region code
+use anchor_lang::prelude::*;
+
+declare_id!("671JxDuZAWcUMMiGbq5YUDUXGBcyAB3tuDDQu7RHUA6h");
+
+#[program]
+pub mod basic_5 {
+    use super::*;
+
+    #[state]
```

Comment:
Note that `#[state]` is deprecated and so we should move this to another example. Preferably in the `tests/` dir.